### ISSUE TITLE
Lockwise sections are refreshed when multitasking to the device home screen and back to Lockwise

### ISSUE THREAD

**Issue by abodea** [mozilla-lockwise/lockwise-android#789]
## Steps to reproduce
1. Launch Lockwise.
2. Login with valid credentials.
3. Open Settings(or any menu).
4. Lock or let the screen for auto-lock.
5. Unlock the phone screen.
### Expected behavior
When unlocking the phone screen, the latest menu should be displayed.
### Actual behavior
When unlocking the phone screen, `All entries` screen is displayed like the app is refreshed.
### Device & build information
* **`Samsung Galaxy Note 9(Android 8.1.0)`**
*  **`Lockwise version 1.1.2 (Build 4342) `**
### Notes
Attachments: [Video](https://drive.google.com/file/d/1l7BxIAGthE2GkiiZsYFgaEprrFu16oKW/view?usp=sharing)

**Comment by eliserichards** [comment]
This is expected behavior. Do we want to change this @sandysage @changecourse?

**Comment by changecourse** [comment]
Yes, I think we should change this... but as a future feature to consider maintaining state (even if this screen is locked) and consider some persistence for a (yet to be determined) period of time, then clear the persistence and revert to the login list.
